web/src/scripts/webflow/sectorSync.mjs

```diff
@@ -66,7 +66,7 @@ const getUnUsedSectors = async () => {
   const current = await getCurrentSectors();
   const overLap = getOverlappingSectorsFunc(current);
   return current.filter((item) => {
-    return !(overLap.includes(item) || item._id == allIndustryId);
+    return !(overLap.includes(item) || item.id == allIndustryId);
   });
```

Comment:
I updated these keys as per version 2 of the Webflow API. The documentation link is in the description, along with a complete list of changes.